docs/unsettled-fees.md

```diff
@@ -0,0 +1,77 @@
+# Unsettled Fees System
+
+## Overview
+
+Our protocol charges fees for order execution, not order placement. This is achieved through dynamic fee calculation using an "unsettled fees" system.
+
+## How Dynamic Fee Calculation Works
+
+When a user places an order, we follow this process:
+
+1. **Place the order** in the DeepBook
+2. **Analyze the order execution**:
```

Comment:
Mb additionally we could share a simple example when we'll charge what?

E.g. for market order, if it would be fully taker, and for the limit, which might be some % taker, some % maker